csug/syntax.stex

```diff
@@ -741,10 +766,9 @@ just that.
 \schemedisplay
 (define-syntax get-property
   (lambda (x)
-    (lambda (r)
-      (syntax-case x ()
-        [(_ id key)
-         #`'#,(datum->syntax #'* (r #'id #'key))]))))
+    (syntax-case x ()
+      [(_ id key)
+       #`'#,(datum->syntax #'* (property-value #'id #'key))])))
```

Comment:
Please keep the original code, because it demonstrates how to use the lookup procedure. Add a note:

> This macro can also be defined without the lookup procedure using \scheme{property-value} as in the \scheme{get-info} example above.

Reply:
This is related to my comment above; if we agree that new code probably wants to use `property-value`, the primary example should be formulated with it. I could add the old version as a secondary example.

Reply:
It's fine with me to have get-info use the lookup procedure and get-property use property-value. I want to have an example of each way.